cli/cmd/testdata/upgrade_default.golden

```diff
@@ -1653,7 +1653,7 @@ spec:
         - 4190,4191
         - --outbound-ports-to-ignore
         - "443"
-        image: gcr.io/linkerd-io/proxy-init:TEST-VERSION
+        image: gcr.io/linkerd-io/proxy-init:dev-undefined
```

Comment:
related to other comment, i'd expect these to become `UPGRADE-CONTROL-PLANE-VERSION`. seeing `dev-undefined` makes me assume this field was never set.